episodes/02-setup.md

```diff
@@ -1,39 +1,93 @@
 ---
 title: Setting Up Git
-teaching: 5
+teaching: 10
 exercises: 0
 ---
 
 ::::::::::::::::::::::::::::::::::::::: objectives
 
 - Configure `git` the first time it is used on a computer.
-- Understand the meaning of the `--global` configuration flag.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
 :::::::::::::::::::::::::::::::::::::::: questions
 
 - How do I get set up to use Git?
```

Comment:
Would "How do I set up Git" be a simpler phrasing?

Reply:
Edited in https://github.com/epiverse-trace/git-rstudio-basics/pull/13/commits/cb3f384b89dc6fe9285f1a4de9434057b76216f3